.changes/v2.13.0/392-bug-fixes.md

```diff
@@ -0,0 +1,2 @@
+* Fix Issue #390: `catalog.Delete()` ignores task [GH-392]
```

Comment:
Please add a bit more explanation of what are fixing here, so for a new reader it's clear from this line.

Reply:
Isn't the reference to the issue – where the problem is explained in detail – not sufficient?

Reply:
I want to make it instantly clear for the users without the need to click.

Suggestion:

"Fix Issue #390: `catalog.Delete()` ignores returned task and responds immediately which could have caused failures [GH-392]

Inspired by:

"Catalog deletion function does not take into account returned task and responds immediately while catalog deletion task is running and may even fail.

This needs to be fixed in order to properly track catalog deletion and catch any errors."

Reply:
Changed